### ISSUE TITLE
Undocumented Zeitwerk dependency

### ISSUE THREAD

**Issue by wusher** [joeldrapper/literal#3]
Running literal without zeitwerk installed raises the error: 
```
.rbenv/versions/3.2.1/lib/ruby/gems/3.2.0/gems/literal-0.1.0/lib/literal.rb:6:in `<module:Literal>': uninitialized constant Literal::Zeitwerk (NameError)

  Loader = Zeitwerk::Loader.for_gem.tap(&:setup)
```

Sample:  
```ruby 
require "bundler/inline"

gemfile do
  source "https://rubygems.org"
  gem "literal"
end

class Address < Literal::Struct
  attribute :street, String
  attribute :city, String
  attribute :state, String
end
```
save that in a file called `main.rb`  a run `ruby main.rb` 

Added `gem 'zeitwork'` to the gem file fixes it : 
```ruby 
gemfile do
  source "https://rubygems.org"

  gem 'zeitwerk', '~> 2.4.2'
  gem "literal"
end
```

**Comment by joeldrapper** [comment]
Thanks @wusher. It's very early days for the gem, I pushed a version to RubyGems to reserve the name. I’ll fix the Zeitwerk dependency now.

**Comment by joeldrapper** [comment]
Should be fixed by this commit `991199cb38c44a5328151645e0d068bfde52ed35`. I’m not going to push any more releases to Rubygems until the API is more stable and it's ready for wider beta testing. I really appreciate you taking it for a spin and sending feedback. 🙏
